### PR TITLE
Restoring changes to build with ament_cmake

### PR DESCRIPTION
**Pull request by anasarrak**

### CODE REVIEW

**Review by LanderU — CHANGES_REQUESTED** [review]
Error(s):
- The manifest of package "octomap" (with format version 3) must not contain the following tags: run_depend
- Please replace <run_depend> tags with <exec_depend> tags.

**Comment by anasarrak** [comment]
@LanderU  Changed on https://github.com/AcutronicRobotics/octomap/pull/3/commits/423814e75432c4616890c5dd6cd5ee8b9eb2d205

**Review by LanderU — CHANGES_REQUESTED** [review]
https://github.com/anasarrak/octomap/blob/devel/octovis/package.xml#L21-L29

This is redundant.

File: octovis/package.xml [hunk 1]
```diff
@@ -17,12 +17,7 @@
 
   <exec_depend>ament_cmake</exec_depend>
```
Comment (LanderU) [inline octovis/package.xml]: Can we push down this?

Comment (anasarrak) [reply octovis/package.xml]: Is not needed, removed here https://github.com/anasarrak/octomap/commit/6e7b57f55d93ad560cd1f036cc4de7f2b53f2720, with buildtool_depend sould be enough

File: octomap/package.xml [hunk 2]
```diff
@@ -14,7 +14,8 @@
   <url type="website">http://octomap.github.io</url>
   <url type="bugtracker">https://github.com/OctoMap/octomap/issues</url>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <run_depend>ament_cmake</run_depend>
```
Comment (LanderU) [inline octomap/package.xml]: Change this

**Pull request closed without merging**